firebase-firestore/src/main/java/com/google/firebase/firestore/local/LocalDocumentsView.java

```diff
@@ -69,6 +71,28 @@ private MaybeDocument getDocument(DocumentKey key, List<MutationBatch> inBatches
     return document;
   }
 
+  @Nullable
+  private List<MaybeDocument> getDocumentsInternal(Iterable<DocumentKey> keys, List<MutationBatch> batches) {
+    List<MaybeDocument> documents = remoteDocumentCache.getAll(keys);
+    // TODO(varconst): uncomment and fix.
```

Comment:
I didn't bother with batches, because there are no pending write batches in the case being tested.